code/controllers/subsystem/movement/movement_types.dm

```diff
@@ -25,6 +25,8 @@
 	var/timer = 0
 	///Is this loop running or not
 	var/running = FALSE
+	///Disables movement without messing with the timer system
+	var/blocked = FALSE
```

Comment:
This is a waste, you'll end up having the loop firing constantly without doing anything.